modules/FhirAPI/src/FhirAPI/Service/FhirRequestParamsHandler.php

```diff
@@ -34,7 +34,8 @@ class FhirRequestParamsHandler
                                               '_total',
                                               '_elements',
                                               '_contained',
-                                              '_containedType'
+                                              '_containedType',
+                                              'filter'
```

Comment:
What the meaning that "filter" is in this list? is it removed from the regular query parameters? 
I'm not sure there aren't other resources with "filter" in the query string

I think it needs to be special parameter only for READ calls and nor for search

Reply:
it is not removed just handled if you know where we use filter I can check it is not hurt

Reply:
![image](https://user-images.githubusercontent.com/20621444/87520291-6c6c1d80-c68b-11ea-8b5b-2a2ea1b0b572.png)

Reply:
for others we can use "_filter"   about just filter I don't know it is not in any list.